libcomps/src/python/src/pycomps_categories.c

```diff
@@ -116,85 +116,6 @@ PyObject* PyCOMPSCat_repr(PyObject *self) {
     return ret;
 }
 
-int PyCOMPSCat_print(PyObject *self, FILE *f, int flags) {
```

Comment:
See commit msg about ```tp_print``` slot being deprecated in Python 3, prior to it being re-used for an entirely different purpose now.

According to the Python 2 documentation this function should never output different data than what ```tp_str``` / ```tp_repr``` (which is why it was deprecated) and so I'm assuming that it can just be removed.  It doesn't look like anyone has complained about functionality not working on Python 3.